brutils/formatpis.py

```diff
@@ -0,0 +1,58 @@
+
+
+# OPERATIONS
+############
+
+
+def validate(pis): # type: (str) -> bool
+    """
+    Validate PIS number.
+    Based on rules available in https://www.macoratti.net/alg_pis.htm
+    If PIS is valid return True or False if PIS number is invalid.
+    """
+    
+    if not pis.isdigit() or len(pis) != 11:
+        return False
+
+    return pis[-1] == str(_checksum(pis))
+    
+
+def is_valid(pis):  # type: (str) -> bool
+    """
+    Evaluates that pis is String and calls validate.
+    """
+
+    return isinstance(pis, str) and validate(pis)
+
+
+def _checksum(base_pis): # type: (str) -> int
+    """
+    Calculates the checksum digit of the given `base_pis` string.
+    The checksum is calculate based on rules described here https://www.macoratti.net/alg_pis.htm
+    """
+    multipliers = [3, 2, 9, 8, 7, 6, 5, 4, 3, 2]
+    digits = [base_pis[:-1][i:i+1] for i in range(0, len(base_pis[:-1]), 1)]
+    digit_sum = sum(int(digit) * multiplier 
+                     for digit, multiplier in zip(digits, multipliers))
+    check_digit = 11 - (digit_sum % 11)
+    
+    if check_digit in [10, 11]:
+        return 0
+    return check_digit
+
+# FORMATTING
+############
+
+def format_pis(pis):  # type: (str) -> str
+    """
+    Format an adequately formatted numbers-only PIS string,
+    Returns a PIS formatted with standard visual aid symbols.
+    Returns None if PIS is invalid.
+    """
+
+    if not is_valid(pis):
+        return None
+
+    return "{}.{}.{}-{}".format(pis[:3], pis[3:8], pis[8:10], pis[10:11])
```

Comment:
```suggestion
def format_pis(pis):  # type: (str) -> str
    """
    Format an adequately formatted numbers-only PIS string,
    Returns a PIS formatted with standard visual aid symbols.
    Returns None if PIS is invalid.
    """

    if not is_valid(pis):
        return None

    return "{}.{}.{}-{}".format(pis[:3], pis[3:8], pis[8:10], pis[10:11])
```
@patricia-salles a função está correta, a não ser por uma coisa:
1 - ela deve seguir o padrão dos outros utilitários, e estar no arquivo chamado `pis.py` - que é onde estão todas as outras funções deste assunto.. o mesmo acontece com os testes: todos os testes dos utilitários de PIS ficam em `test_pis.py`

Reply:
Oi @antoniamaia, realizei as alterações que você comentou e a documentação no README.